properties_pane/field_level/fieldLevelConfig.json

```diff
@@ -6405,6 +6405,7 @@ making sure that you maintain a proper JSON format.
         ]
       },
       {
+        "fieldName": "Min Properties",
```

Comment:
It is not required, the name is taken from app config. And it doesn't follow convention, we use `property{sth}` not `field{sth}`. I see that it was written this way, but it shouldn't